docs/dashboards/numeric-component.md

```diff
@@ -0,0 +1,34 @@
+title: Numeric Component
+description: Shows numeric values
+
+Displays the value of the selected metric. The displayed values depends on the selected data time picker located at the top right of the dashboard.
+
+![Numeric Component](../images/dashboards/numeric-component.png)
+
+## Configuration
+
+You can configure the componenteither during its initial creation or afterward by clicking on the three dots that appear on hover and selecting 'Edit Component'.
+
+- Choose All Apps or a specific App that you want to fetch data from.
+- Choose one or more metrics metric
+- Enter optional parameters such as:
+  - [Transformation](https://sematext.com/docs/dashboards/chart-builder/#transformation)
+  - [Filter By](https://sematext.com/docs/dashboards/chart-builder/#filter-by)
+  - [Group by](https://sematext.com/docs/dashboards/chart-builder/#group-by)
+  - [Rollup By](https://sematext.com/docs/dashboards/chart-builder/#rollup-by)
+    
```

Comment:
Will we want to explain "Only latest interval"?
I see we have a super short tooltip for it in the UI.  If we describe it here we could link to it from there?

Reply:
I will explain it once we have the correct logic. https://sematext.atlassian.net/browse/SC-17236
Not only for this one, we have only latest interval option in other components too, I'll add a section for this in all of them.